chispa/__init__.py

```diff
@@ -1,37 +1,9 @@
 from __future__ import annotations
 
-import os
-import sys
-from glob import glob
-from typing import Callable
+from collections.abc import Callable
 
 from pyspark.sql import DataFrame
 
-# Add PySpark to the library path based on the value of SPARK_HOME if pyspark is not already in our path
```

Comment:
Do some runtimes need this?

Reply:
I don't think so. PyPspark contains all the JARs.